agent/rpc/peering/service.go

```diff
@@ -679,9 +668,9 @@ func (s *Server) getExistingPeering(peerName, partition string) (*pbpeering.Peer
 //
 // We define a DIALER as a peering that has server addresses (or a peering that is created via the Establish endpoint)
 // Conversely, we define an ACCEPTOR as a peering that is created via the GenerateToken endpoint
-func validatePeer(peering *pbpeering.Peering, allowedToDial bool) error {
-	if peering != nil && peering.ShouldDial() != allowedToDial {
-		if allowedToDial {
+func validatePeer(peering *pbpeering.Peering, shouldDial bool) error {
+	if peering != nil && peering.ShouldDial() != shouldDial {
+		if shouldDial {
```

Comment:
By aligning the var name with `peering.ShouldDial` it's easier to understand at a glance